aten/src/ATen/native/RNN.cpp

```diff
@@ -293,10 +293,10 @@ static std::vector<QuantizedCellParamsDynamic> gather_quantized_params_dynamic(
     auto bias_hh = packed_struct_hh.bias.value_or(undefined);
     result.emplace_back(params[i], params[i + 1], bias_ih, bias_hh);
   }
-  return result;
 #else // USE_FBGEMM
   TORCH_INTERNAL_ASSERT(false, "Tried to use quantized RNN wihtout FBGEMM!")
 #endif // USE_FBGEMM
+  return result;
```

Comment:
What's the reason for this change? Is this to workaround some no-return-statement lint issue?

Reply:
@jamesr66a , thanks for the review. I added that shut off the compiler since there has to be an unconditional return variable of type std::vector<QuantizedCellParamsDynamic> regardless of USE_FBGEMM macro.